main.go

```diff
@@ -255,7 +260,11 @@ func wrapMain() error {
 
 		f.Close()
 
-		pathArg = append(pathArg, outPaths[t])
+		if t == vargs.SecretTemplate {
```

Comment:
Do you mind using the same logic as L231 (check against Template).

Reply:
updated